tasks/opsep/update-l2pao-to-aliased-l1pao/README.md

```diff
@@ -0,0 +1,39 @@
+# Op Sepolia - Update L2ProxyAdmin owner to be aliased L1ProxyAdmin owner address
+
+Status: [DRAFT]()
+
+## Objective
+
+This is the playbook for executing Upgrade #X, `set(L2PAO, aliased(L2PAO))`, on Op Sepolia (chain ID 11155420).
```

Comment:
Nit, to just avoid using nonstandard acronyms before defining them + avoids using pseudocode that doesn't map to the contracts.
```suggestion
This is the playbook for executing Upgrade #X, to update the L2ProxyAdmin owner (L2PAO) to be the aliased L1ProxyAdmin owner (L1PAO) address, on OP Sepolia (chain ID 11155420).
```